src/components/Navbar/SettingsDrawer/WordByWordSection.tsx

```diff
@@ -62,6 +58,33 @@ const WordByWordSection = () => {
 
   const wordByWordFontScale = useSelector(selectWordByWordFontScale, shallowEqual);
 
+  const [hasError, setHasError] = useState(false);
+  const [translations, setTranslations] = useState<AvailableWordByWordTranslation[]>([]);
+
+  useEffect(() => {
+    getAvailableWordByWordTranslations(lang)
+      .then((res) => {
+        if (res.status === 500) {
+          setHasError(true);
+        } else {
+          const data = uniqueArrayByObjectProperty(res.wordByWordTranslations, 'isoCode');
+          setTranslations(data);
+        }
+      })
+      .catch(() => {
+        setHasError(true);
+      });
+  }, [lang]);
+
+  if (hasError) {
+    return null;
```

Comment:
I think we need to ask what should we show in this case because not showing anything at all in-case of an error is not ideal. I suggest we allow fetch retrying by using `DataFetcher` component.